Realm/RLMObjectSchema.mm

```diff
@@ -188,6 +199,24 @@ + (NSArray *)propertiesForClass:(Class)objectClass isSwift:(bool)isSwiftClass {
         }
     }
 
+    if (NSArray *optionalProperties = [objectUtil getOptionalPropertyNames:swiftObjectInstance]) {
```

Comment:
I'd prefer removing stub implementations when they're not used, and definitely think it's clearer to conditionally call the appropriate helpers. But this conversation certainly shouldn't prevent a null beta release.